appeals/api/src/server/endpoints/integrations/integrations.controller.js

```diff
@@ -0,0 +1,31 @@
+import { createAppeal, createDocument } from '#repositories/integrations.repository.js';
+import { mapAppealFromTopic, mapDocumentFromTopic } from './integrations.mapper.js';
+
+/** @typedef {import('@pins/appeals/index.js').BlobInfo} BlobInfo */
+/** @typedef {import('@pins/appeals/index.js').AddDocumentsResponse} AddDocumentsResponse */
+/** @typedef {import('@pins/appeals.api').Schema.Folder} Folder */
+/** @typedef {import('express').RequestHandler} RequestHandler */
+
+/**
+ * @type {RequestHandler}
+ * @returns {Promise<any>}
+ */
+export const postAppealSubmission = async (req, res) => {
+	const data = mapAppealFromTopic(req.body);
+	const result = await createAppeal(data);
+
+	// @ts-ignore
+	return res.send(result);
+};
+
+/**
+ * @type {RequestHandler}
+ * @returns {Promise<any>}
```

Comment:
This method could have a return type of `Promise<Response>` and then the `'@ts-ignore` wouldn't be needed below.

`Response` would need have a typedef like `/** @typedef {import('express').Response} Response */`